docs/diagnostics/PX1007.md

```diff
@@ -15,13 +15,28 @@ The code fix does one of the following:
  - Adds the `summary` XML tags, in which you type the description of the code item
  - Adds the `exclude` XML tag, which excludes the item from the API and DAC References
 
-The new diagnostic is available only if the value of **Tools > Options > Acuminator > Code Analysis > Enable PX1007 diagnostic** is set to _True_. By default, the value of this option is _False_. 
-
-This diagnostic is intended for Acumatica ERP developers, however it can be used by any developers to provide documentation of source code.
-
-The diagnostic is currently displayed for all public entities (classes, interfaces, structs, enumerations, and delegates) and DAC property fields no matter in which namespace they are located. The diagnostic is not displayed for other fields, properties, and methods of public entities. 
-
-The diagnostic is not displayed for child classes if their parent class has the `exclude` XML tag.
+The new diagnostic is available only if the value of **Tools > Options > Acuminator > Code Analysis > Enable PX1007 diagnostic** is _True_. By default, the value of this option is _False_. 
+
+This diagnostic is intended for Acumatica ERP developers; however, it can be used by any developer to provide documentation of the source code.
+
+The diagnostic is currently displayed for the following public items:
+ - Classes that are not DACs 
+ - DACs with the `PXCacheName` attribute
+ - DAC property fields
+ - Interfaces
+ - Structs
+ - Enumerations
+ - Delegates
+
+The diagnostic is not displayed for the following items: 
+ - Fields, properties, and methods of public entities (except DAC property fields).
+ - Child classes if their parent class has the `exclude` XML tag.
+ - DACs with the `PXHidden`, `PXInternalUseOnly`, or `Obsolete` attribute.
+ - Class fields of DACs. In DACs, XML annotations should be specified only for property fields.
+ - System fields of DACs, such as `CreatedDateTime` or `LastModifiedDateTime`.
+ - Public items from the assemblies whose names contain `Test` or `Benchmark`.
```

Comment:
I think that it is not a specific trait of PX1007 diagnostic - it can be applied to all of them.
Therefore, I would place it in some general place like Summary or DiagnosticSuppression files

Reply:
Summary is definitely not the right place, as well as DiagnosticSuppression. This is written in Release Notes. I can just remove it here.

Reply:
I'm not sure - maybe at least it should be in the readme. Or DiagnosticSuppression. 
Imho, it is wrong to read all release notes just to find out description of product behavior because it is missing in all other places. 
This is somewhat important information that some assemblies won't be analyzed. If I was a user of Acuminator I would definitely want to know more about the rules of exclusion

Reply:
We don't have full documentation for Acuminator and we do not pretend to write it...

Reply:
Yes, we don't document the API and most of implementation details but we try to document the functionality and this info is related to functionality.  So, it will be great if we can add a small text about it to the summary

Reply:
So far we do not document the UI as well (except in Release Notes).
Text about enable code analysis switch and the excluded unit test projects added to Summary.md

Reply:
Maybe we should write the detailed docs for UI at some point. Anyway, thanks a lot!